packages/core/src/actions/send.ts

```diff
@@ -99,23 +99,39 @@ export function send<
           : params.delay;
       }
 
-      let resolvedTarget = isFunction(params.to)
+      const resolvedTarget = isFunction(params.to)
         ? params.to(ctx, _event.data, meta)
         : params.to;
-      resolvedTarget =
-        isString(resolvedTarget) &&
-        resolvedTarget !== SpecialTargets.Parent &&
-        resolvedTarget !== SpecialTargets.Internal &&
-        resolvedTarget.startsWith('#_')
-          ? resolvedTarget.slice(2)
-          : resolvedTarget;
+      let targetActorRef: AnyActorRef | undefined;
+
+      if (typeof resolvedTarget === 'string') {
+        if (resolvedTarget === SpecialTargets.Parent) {
+          targetActorRef = actorContext?.self._parent;
+        } else if (
+          resolvedTarget === SpecialTargets.Internal ||
+          // SCXML compatibility
+          resolvedTarget === '#_internal'
+        ) {
```

Comment:
`SpecialTargets.Internal` already equals to `#_internal` so there is no need for that extra branch
```suggestion
        } else if (resolvedTarget === SpecialTargets.Internal) {
```